examples/vpc-flow-logs /vpc-flow-logs.tf

```diff
@@ -0,0 +1,15 @@
+provider "aws" {
+}
+
+module "vpc-flow-logs" {
+  source = "coralogix/aws/coralogix//modules/vpc-flow-logs"
+
+  coralogix_region   = "Europe"
+  custom_domain      = "https://<your custom doamin>/api/v1/logs"
```

Comment:
If you provide the region why need your custom domain

Reply:
@guyrenny oded ask why in the example you put both coralogix_region and custom_domain
since if you have custom_domain the region is not taken

Reply:
but if the user will not use the custom_domain and leave it as blank i will need to leave a default option, which is the custom_region, but if want to use the custom_domain i want him to see the format that he needs to use.